test pr

pandinocoder opened the pull request.

lodicolo commented:
/helptest

lodicolo commented:
/helptest

lodicolo commented:
Developers have requested community assistance to test the code changes in this pull request.

If you'd like to help you can download binaries for this PR [here](https://github.com/lodicolo/Intersect-Engine/actions/runs/6451804861).

These builds are only meant for testing to validate bug fixes and could cause significant damage to your game data. In other words, make a backup of your game data before proceeding.

Please comment below to provide testing feedback. Thanks!

lodicolo commented:
/helptest

lodicolo commented:
Developers have requested community assistance to test the code changes in this pull request.

If you'd like to help you can download binaries for this PR [here](https://github.com/lodicolo/Intersect-Engine/actions/runs/6463030306).

These builds are only meant for testing to validate bug fixes and could cause significant damage to your game data. In other words, make a backup of your game data before proceeding.

Please comment below to provide testing feedback. Thanks!

pandinocoder committed:
workflow fixes/improvements
- made pull_request have a manual dispatch
- delete request_help because it's now redundant
  - removed it because I couldn't get the thing to stop failing at the end
- made arifacts upload on failure

pandinocoder committed:
made prod certs default self-signed (since most people will only ever run it in prod they wouldn't know about this feature)
